### PR TITLE
ash-13-1-python

### PR DESCRIPTION
**Pull request by ash-13**

### CODE REVIEW

File: Q1/ash-13/Solution.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Python3 implementation of the approach  
+
```
Comment (Yatharth0045) [inline Q1/ash-13/Solution.py:2]: Please add `run.md`
Go through the README of this repo for more instructions regarding PR submission. @ash-13

File: Q1/ash-13/Solution.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  
+    # If the digit sum is palindrome  
+    if (isPalindrome(sum)) : 
+        return True;  
+    return False;  
+  
+# Driver code  
+if __name__ == "__main__" :  
+  
+    n = 56;
```
Comment (Yatharth0045) [inline Q1/ash-13/Solution.py:58]: Take user input instead of  hard-coding the value @ash-13

**Review by Yatharth0045 — CHANGES_REQUESTED** [review]

File: Q1/ash-13/Solution.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    sum = 0;  
+    while (n > 0) : 
+        sum += (n % 10);  
+        n //= 10;  
+  
+    return sum;  
+  
+# Function that returns true  
+# if n is palindrome  
+def isPalindrome(n) :
```
Comment (Yatharth0045) [inline Q1/ash-13/Solution.py:16]: Well Commented code 👍🏻  @ash-13

File: Q1/ash-13/Solution.py (hunk 1 above)
Comment (ash-13) [reply Q1/ash-13/Solution.py:2]: Sure bhai

**Pull request merged**